src/test/resources/template/ParameterReuseRecipe$1_before.java

```diff
@@ -16,7 +16,19 @@
 package foo;
 import org.openrewrite.java.*;
 
+/**
+ * OpenRewrite `before` template created for  `foo.ParameterReuseRecipe$1`.
+ */
 public class ParameterReuseRecipe$1_before {
+    /**
+     * Instantiates a new {@link ParameterReuseRecipe$1_before} instance.
+     * @return the new instance
```

Comment:
`@return` doesn't apply (or at least, isn't necessary) for constructors.

Reply:
Thanks! Removed.